Hymns/Display/HymnLyricsViewModel.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Resolver
+
+class HymnLyricsViewModel: ObservableObject {
+    
+    @Published var lyrics: [Verse]? = [Verse]()
+    
+    @Injected private var hymnsRepository: HymnsRepository
+    
+    init() {
+        hymnsRepository.getHymn(hymnIdentifier: HymnIdentifier(hymnType: .classic, hymnNumber: "1151")) { hymn in
+            guard let hymn = hymn, !hymn.lyrics.isEmpty else {
+                self.lyrics = nil
+                return
+            }
+            
+            self.lyrics = hymn.lyrics
+        }
+    }
+}
+
+extension Resolver {
```

Comment:
How does this work?

Reply:
Here is how extensions work: https://www.hackingwithswift.com/sixty/9/3/extensions

We are extending the Resolver class to have extra methods that essentially "registers" our dependencies, so they can be available via the resolve() method and the @Injected parameter.